test-suite/misc/bench-render/result.html

```diff
@@ -0,0 +1,103 @@
+<html>
```

Comment:
![image](https://user-images.githubusercontent.com/1253341/226421705-71d3448c-d628-4912-b973-ed60c0e5d9ca.png)

Line 9 looks funny.  Should the bottom half be tan?